packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -428,19 +428,19 @@ export class ExecutionEngineHttp implements IExecutionEngine {
 
     switch (newState) {
       case ExecutionEngineState.ONLINE:
-        this.logger.info("ExecutionEngine became online");
+        this.logger.info("Execution client became online");
         break;
       case ExecutionEngineState.OFFLINE:
-        this.logger.error("ExecutionEngine went offline");
+        this.logger.error("Execution client went offline");
         break;
       case ExecutionEngineState.SYNCED:
-        this.logger.info("ExecutionEngine is synced");
+        this.logger.info("Execution client is synced");
```

Comment:
```suggestion
        this.logger.info("Execution client is synced and client can now participate fully in the network");
```

Reply:
How long does it take usually for Lodestar to know whether the EL is actually in a `synced` state or if this is just the default `online` state due to beacon startup? If there's some sort of way to delay/timeout this message to ensure it actually is `synced`, that might work better?

Reply:
> How long does it take usually for Lodestar to know whether the EL is actually in a synced state

After first request to EL we know the actual status and print out a status change message

> ensure it actually is synced

We know the correct state at this point in time since we made an request to EL, "can now participate" is just unnecessary here because beacon node was just starting up and EL was likely always synced.
The message only makes sense to me if EL goes from syncing to synced.